source/plugins/wakatime/index.mjs

```diff
@@ -1,44 +1,56 @@
 //Setup
-  export default async function({login, q, imports, data, account}, {enabled = false, token} = {}) {
-    //Plugin execution
-      try {
-        //Check if plugin is enabled and requirements are met
-          if ((!enabled)||(!q.wakatime))
-            return null
+export default async function ({ login, q, imports, data, account },{ enabled = false, token } = {}) {
+  //Plugin execution
+  try {
+    //Check if plugin is enabled and requirements are met
+    if (!enabled || !q.wakatime) return null;
 
-        //Load inputs
-          let {sections, days, limit} = imports.metadata.plugins.wakatime.inputs({data, account, q})
-          if (!limit)
-            limit = void(limit)
-          const range = {"7":"last_7_days", "30":"last_30_days", "180":"last_6_months", "365":"last_year"}[days] ?? "last_7_days"
+    //Load inputs
+    let {
+      sections,
+      days,
+      limit,
+      apiurl,
+      wakapi = false,
+      user = "current",
```

Comment:
No need to set defaults on these, they're automatically set by `imports.metadata.plugins.wakatime.inputs` depending on what is defined in `metadata.yml` 👍